docs/src/site/twiki/EntitySpecification.twiki

```diff
@@ -416,7 +420,12 @@ waiting for the feed instance, parallel decides the concurrent replication insta
 order decides the execution order for replication instances like FIFO, LIFO and LAST_ONLY.
 DistCp options can be passed as custom properties, which will be propagated to the DistCp tool. "maxMaps" represents
 the maximum number of maps used during replication. "mapBandwidth" represents the bandwidth in MB/s
-used by each mapper during replication. "overwrite" represents overwrite destination during replication.
+used by each mapper during replication.
+"mapMemory" represents the mapreduce map memory in mb to be specified to the respective replication and retention jobs.
```

Comment:
nit: just to keep it consistent either start it from the same line or move every property name to a separate line.
Right now mapBandwith is in one line and rest of setting is in form of paragraph.